### PR TITLE
Fix for #2249; Add handling of REFRESH command and introduce lastMoti…

### PR DESCRIPTION
**Pull request by hmerk**
…onDetected channel for MotionSensor

Signed-off-by: Hans-Jörg Merk <hans-joerg.merk@t-online.de>

### CODE REVIEW

**Comment by maggu2810** [comment]
@hmerk Are you aware of the build failure?

```text
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:0.23.1:compile (default-compile) on project org.eclipse.smarthome.binding.wemo: Compilation failure: Compilation failure:
[ERROR] /home/travis/build/eclipse/smarthome/extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/handler/WemoHandler.java:[330]
[ERROR] updateState(CHANNEL_LASTMOTIONDETECTED, lastMotionDetected);
[ERROR] ^^^^^^^^^^^^^^^^^^^^^^^^^^
[ERROR] CHANNEL_LASTMOTIONDETECTED cannot be resolved to a variable
[ERROR] 1 problem (1 error)
[ERROR] -> [Help 1]
```

**Comment by hmerk** [comment]
Thanks @maggu2810 , thats strange cause the build in my IDE showed no error. I will have a look into it later tonight.

**Comment by hmerk** [comment]
Argh, I forgot to include the updated WemoBindingConstants into the PR. Will add it tonight.

**Comment by hmerk** [comment]
Hmm, I can't find any hint in the build log that the WeMo Binding failed, so don't know what went wrong.
@maggu2810 could you check the log please.

**Comment by maggu2810** [comment]
Hm

```text
[INFO] Downloaded: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/xtend/org.eclipse.xtend.core/maven-metadata.xml (402 B at 2.9 KB/sec)


No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

The build has been terminated
```

I will retrigger the build, perhaps something went wrong on fetching the dependencies.

**Comment by hmerk** [comment]
Ok, so now all checks are passed. Thanks @maggu2810

File: extensions/binding/org.eclipse.smarthome.binding.wemo/ESH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -144,6 +145,13 @@
         <state readOnly="true"></state>
     </channel-type>
     
+     <channel-type id="lastMotionDetected" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>Last Activity (Date/Time)</label>
+        <description>Date/time when last motion was detected</description>
+        <category>Date</category>
```
Comment (maggu2810) [inline extensions/binding/org.eclipse.smarthome.binding.wemo/ESH-INF/thing/thing-types.xml]: @kaikreuzer IIRC you complained using category Switch for item-type Switch. What about using category Date for DateTime item-type?

Comment (kaikreuzer) [inline extensions/binding/org.eclipse.smarthome.binding.wemo/ESH-INF/thing/thing-types.xml]: Well, Date isn't even listed [here](https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#channel-categories), therefore it is clearly wrong to put it here.

Comment (hmerk) [inline extensions/binding/org.eclipse.smarthome.binding.wemo/ESH-INF/thing/thing-types.xml]: Hm, as there is no item of type DateTime listed at all, what category would you suggest.

Comment (hmerk) [inline extensions/binding/org.eclipse.smarthome.binding.wemo/ESH-INF/thing/thing-types.xml]: OK, removed the category for the two channels of DateTime type.

**Pull request merged**